Mobile/starter_project_matador/lib/features/example/presentation/screen/splash_screen.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:matador/features/example/presentation/widgets/background_container.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    return Container(
+      color: const Color.fromARGB(255, 233, 233, 233),
+      child: Stack(
+        children: [
+          BackgroundContainer(top: 200, left: 300),
```

Comment:
> Avoid giving size this way. It would be better if you do the math using the screen's width and height.

Yes, you are right but it has no significant effect in this case.

Reply:
> Avoid giving size this way. It would be better if you do the math using the screen's width and height.

Yes, you are right but it has no significant effect in this case.

Reply:
if there is no significant effect then we should always do as the flutter guideline suggests.